File: myblog-server/src/main/java/com/site/blog/context/UserContextHolder.java
package com.site.blog.context;

import cn.hutool.extra.spring.SpringUtil;


/**
 * @author yanni
 * @date time 2022/6/18 3:50
 * @modified By:
 */
public class UserContextHolder {
    public static UserContext me() {
        return SpringUtil.getBean(UserContext.class);
    }

}

File: myblog-jpa/src/main/kotlin/com/site/blog/context/UserContextHolder.kt
package com.site.blog.context

import cn.hutool.extra.spring.SpringUtil

/**
 * @author yanni
 * @date time 2022/6/18 3:50
 * @modified By:
 */
object UserContextHolder {
    fun me(): UserContext {
        return SpringUtil.getBean(UserContext::class.java)
    }
}